frontend/src/Admin/components/PortfolioCard.js

```diff
@@ -0,0 +1,234 @@
+import React, { Component, Fragment } from 'react'
```

Comment:
How much of this file was copied from shows card? Is there a way we can use any shared components?
For the student version of this I ended up turning some of the matching internals into components that they both used.

Reply:
Just got it working. Most of it is duplicated. The "shows" card has some additional functionality, I'll take a look at reusing components.